src/api/transactionSubmission/build.ts

```diff
@@ -21,7 +21,17 @@ export class Build {
     this.config = config;
   }
 
-  async transaction(args: {
+  /**
+   * Build a simple transaction
+   *
+   * @param args.sender The sender account address
+   * @param args.data The transaction data
+   * @param args.options optional. Optional transaction configurations
+   * @param args.withFeePayer optional. Whether there is a fee payer for the transaction
+   *
+   * @returns SingleSignerTransaction
+   */
+  async simple(args: {
```

Comment:
Oh, I see... simple is fee payer also, which gets confusing if it's singleSigner, though it does return a SingleSignerTransaction.

Reply:
yea... i'll probably rename that also in a second PR to try and not add 10 more files to this PR